demo/admin/src/common/ContentScopeProvider.tsx

```diff
@@ -32,8 +32,16 @@ export const ContentScopeProvider = ({ children }: Pick<ContentScopeProviderProp
         language: { value: contentScope.language, label: contentScope.language.toUpperCase() },
     }));
 
+    if (!user.allowedContentScopes[0]) {
+        throw new Error("User does not have access to any scopes.");
+    }
+    const defaultValue = {
+        domain: user.allowedContentScopes[0].domain,
+        language: user.allowedContentScopes[0].language,
+    };
```

Comment:
This should probably work as well:

```suggestion
    const defaultValue = user.allowedContentScopes[0];
```

Reply:
Unfortunately not. I get the error:
Type 'ContentScopeInterface' is missing the following properties from type 'ContentScope': domain, languagets(2739)

allowedContentScopes is of the type `ContentScopeInterface[]`.
which is this type:
```
export interface ContentScopeInterface {
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    [key: string]: any;
}
```

This would work:
`const defaultValue = user.allowedContentScopes[0] as ContentScope;`

c67b4af70aa8468bcdd07500cc15c6812fdef52a

Reply:
You could avoid the second type cast by using `userContentScopes` instead of `user.allowedContentScopes`

Reply:
Changed here: 268accd346a8bbc27904f217cd95602c640dbdd9

just checked and I didn't think of that because in the project I made the initial change, the code is still older and the userContentScopes does not exist yet there.